Octokit/GitHubClient.cs

```diff
@@ -100,6 +100,25 @@ public GitHubClient(IConnection connection)
         }
 
         /// <summary>
+        /// Set the timespan to wait before the requests to GitHub api times out.
+        ///
+        /// Exceptions:
```

Comment:
We can drop these Exception details from xmldoc comments... it's not that they aren't informative but just for consistency purposes (since we don't include this detail on any/many other documented calls).  Just `<summary>` and `<param>` entries is fine

Reply:
Ok I will do it.

I think I will replace it by a link towards the `HttpClient.Timeout` doc. What do you think of that?

Or completely remove? (it will be a shame because it could be useful)

Reply:
I push the fix...